idaes/apps/caprese/estimator.py

```diff
@@ -494,9 +494,10 @@ def generate_estimates_at_time(self, t):
         return [val for val in self.vectors.differential[:, t].value]
 
     def load_measurements(self, measured, timepoint=None):
-        super(_EstimatorBlockData, self).load_measurements(
-            measured, "actualmeasurement", timepoint
-        )
+        time = self.time
+        t = timepoint if timepoint is not None else time.first()
```

Comment:
I think it's better to change the argument in the else statement into `time.last()`, because that's the case if timepoint is not given accidentally.